--- a/ccx_2.23/src/printoutnode.f
+++ b/ccx_2.23/src/printoutnode.f
@@ -76,10 +76,10 @@
       elseif((prlab(ii)(1:4).eq.'NT  ').or.
      &       (prlab(ii)(1:4).eq.'TS  ')) then
          if(ithermal(1).le.1) then
-            write(5,'(i10,1x,1p,e13.6)') node,
+            write(5,'(i10,1x,1p,e23.15)') node,
      &           t1(node)
          else
-            write(5,'(i10,1x,1p,e13.6)') node,
+            write(5,'(i10,1x,1p,e23.15)') node,
      &           v(0,node)
          endif
       elseif(prlab(ii)(1:4).eq.'PS  ') then
